ledger/metrics/src/test/suite/scala/com/daml/telemetry/TelemetryContextSpec.scala

```diff
@@ -0,0 +1,62 @@
+// Copyright (c) 2021 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
```

Comment:
Can you add test case(s) for `runInNewSpan` as well, please?

Reply:
Sure, will do in the next PR.